internals/webhooks/capp_webhook.go

```diff
@@ -50,3 +68,12 @@ func (c *CappValidator) handle(ctx context.Context, req admission.Request, capp
 	}
 	return admission.Allowed("")
 }
+
+func (c *CappValidator) getRCSConfig(ctx context.Context) (*rcsdv1alpha1.RCSConfig, error) {
```

Comment:
not sure this function is really needed

Reply:
It is necessary to obtain the RCS config object in the Capp webhook, which is then used to pass the placements for checking site validity in the isSiteValid function. This can be done directly in the handle function, but using a separate function call enhances readability and prevents code duplication.